### PR TITLE
[WFLY-10004] httpclient components moved into wildlfy-core

### PR DESCRIPTION
**Pull request by thaarok**
In https://github.com/wildfly/wildfly-core/pull/3149 was httpclient components moved into wildfly-core.
This PR removes them from wildfly, as they are already part of widlfly-core.

This can be merged without breaking tests only after wildfly-core upgrade!

https://issues.jboss.org/browse/WFLY-10004
https://issues.jboss.org/browse/JBEAP-12340

### CODE REVIEW

**Comment by stuartwdouglas** [comment]
This is ok to test

File: pom.xml [hunk 1]
```diff
@@ -252,7 +249,7 @@
         <version.org.wildfly.build-tools>1.2.9.Final</version.org.wildfly.build-tools>
         <version.org.wildfly.cdi-api-bridge>1.0.1.Final</version.org.wildfly.cdi-api-bridge>
         <version.org.wildfly.checkstyle-config>1.0.5.Final</version.org.wildfly.checkstyle-config>
-        <version.org.wildfly.core>4.0.0.Final</version.org.wildfly.core>
+        <version.org.wildfly.core>5.0.0.Alpha1-SNAPSHOT</version.org.wildfly.core>
```
Comment (stuartwdouglas) [inline pom.xml]: We can't use snapshot deps for Wildfly. If you need a Wildfly core release just leave a comment on the PR and we will hold off merging until the release has been done.

**Review by stuartwdouglas — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (hkalina) [reply pom.xml]: sry, relict of working version, fixed

**Comment by stuartwdouglas** [comment]
Retest this please

**Pull request merged**